packages/cspell-types/src/CSpellSettingsDef.ts

```diff
@@ -409,19 +482,53 @@ export interface BaseSetting {
     noSuggestDictionaries?: DictionaryReference[];
 
     /**
-     * List of RegExp patterns or Pattern names to exclude from spell checking.
+     * List of regular expression patterns or pattern names to exclude from spell checking.
      *
      * Example: ["href"] - to exclude html href.
+     *
+     * By default, several patterns are excluded. See
+     * [the documentation](https://cspell.org/docs/configuration/) for more details.
```

Comment:
The current link is: `[Configuration - CSpell](https://cspell.org/configuration/#cspelljson-sections)`

Reply:
can you offer this as a suggestion so i can click the button "commit suggestion"?